README.md

```diff
@@ -78,3 +78,27 @@ These tests do not require the performance VM environment to run.
      * Marketing product "FooProduct"
      * 1k subscriptions of FooProduct that provide the engineering product 71
    * Each pool will have a source subscription but will not have stacking
+
+ * manual/pool_refresh.jms
+
+    This can be used to profile account refresh.
+      Environment set up by the script
+      * Single owner with 100 pools.
+      * Each pool contains a marketing product that references a single engineering product
+      * The engineering product references a single content set
+
+     Scenarios that are set up for testing with this script
+     * Add a 2nd engineering product & content set to the existing pools.
+     * Add a 2nd content set to the existing engineering product.
+     * Add 100 net new pools
+
+     To use any of these scenarios:
+     1. Deploy a clean candlepin instance with a new database in hosted mode.
```

Comment:
Please add a note indicating that this step (specifically redeploying against a completely fresh db) is necessary for each use of these scenarios!